Dockerfile.ubuntu

```diff
@@ -0,0 +1,91 @@
+# Stage 1: Builder stage
+FROM node:20-bookworm AS build-stage
```

Comment:
Just realized that this is not actually Ubuntu but Debian, at least according to https://hub.docker.com/_/node:
> Some of these tags may have names like bookworm or bullseye in them. These are the suite code names for releases of [Debian⁠](https://wiki.debian.org/DebianReleases) and indicate which release the image is based on.

This is not a problem, just makes the file naming a bit confusing. Sorry, I missed that yesterday!